example-clients/meson.build

```diff
@@ -106,9 +106,14 @@ if build_jack_net
   )
 endif
 
+if has_jackctl_server_create2
```

Comment:
I am sorry, but this is not quiet correct.
This function was introduced in order to fix an issue regarding dbus device reservation, in https://github.com/jackaudio/jack2/commit/f19176657cd8bf3d672554c97872b1f1bc0b7054
So only jack2 >= 1.9.14 has it

That being said, the warning we give regarding missing "jack implementation" prefix on pkg-config files is related, as that requires a recent version of jack2.
So perhaps not a real problem, but something to keep in mind nonetheless

Reply:
Hmm, that detection would fail with jack2 < 1.9.14 then I believe